.github/CONTRIBUTING.md

```diff
@@ -1,12 +1,31 @@
 # How to use Jedis Github Issue
 
-* Github issues SHOULD ONLY BE USED to report bugs, and for DETAILED feature requests. Everything else belongs to the Jedis Google Group.
+* Github issues SHOULD ONLY BE USED to report bugs and for DETAILED feature requests. Everything else belongs to the [Jedis Google Group](https://groups.google.com/g/jedis_redis) or [Jedis GitHub Discussions](https://github.com/redis/jedis/discussions).
 
-Jedis Google Group address:
-  
-https://groups.google.com/forum/?fromgroups#!forum/jedis_redis
+Please post General questions to Google Group or Github discussions. It can be closed without answer when posted to Github issues.
 
-Please post General questions to Google Group. It can be closed without answer when posted to Github issue.
+# How to contribute by Pull Request
```

Comment:
# Contributing to Jedis

Reply:
We consider creating issues as contributions as well. And this block is only about creating PR.